modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2153,24 +2154,27 @@ protected CompletableFuture<Void> handleChangeStableAssignmentEvent(
 
         TablePartitionId tablePartitionId = new TablePartitionId(tableId, partitionId);
 
-        Set<Assignment> stableAssignments = ByteUtils.fromBytes(stableAssignmentsWatchEvent.value());
+        return supplyAsync(() -> {
+            Set<Assignment> stableAssignments = stableAssignmentsWatchEvent.value() == null
```

Comment:
Seems that we may handle null value inside Assignments.fromBytes itself.

Reply:
Yes, I'll do it